cloud_experts_tutorials/cloud-experts-update-component-routes.adoc

```diff
@@ -0,0 +1,252 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-update-component-routes"]
+= Tutorial: Updating component routes with custom domains and TLS certificates
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-update-component-routes
+
+toc::[]
+
+:fn-supported-versions: footnote:[Modifying these routes on {product-title} ROSA versions prior to 4.14 is not typically supported. However, if you have a cluster using version 4.13, you can request for Red Hat Support to enable support for this feature on your version 4.13 cluster by link:https://access.redhat.com/support/cases/new[opening a support case].]
+:fn-term-component-routes: footnote:[We use the term "component routes" to refer to the `OAuth`, `Console`, and `Downloads` routes that are provided when ROSA are first installed.]
```

Comment:
Recommend using quotation marks rather than backticks for literal values in a footnote, as docs.openshift.com does not render them correctly.

Reply:
I don't really think the backticks are necessary. I'm just going to remove them, they don't need to be quoted. I think that came from the original version of this article.